networkpolicy/README.md

```diff
@@ -0,0 +1,79 @@
+# Overview
```

Comment:
Pls provide context and link to Knowledge Center

Reply:
Added